client.go

```diff
@@ -610,7 +610,9 @@ func (c *Client) proxyHTTP(port int) error {
 		}
 
 		buf := new(bytes.Buffer)
-		resp.Write(buf)
+		if err := resp.Write(buf); err != nil {
```

Comment:
I don't think there's much sense in adding code just to make external tools happy.

I think errcheck should be fixed to understand stdlib at least (if not allow for whitelisting) - (*bytes.Buffer).Write [does not return error, ever](https://github.com/golang/go/blob/master/src/bytes/buffer.go#L135).